### DISCUSSION TITLE
Preview Env problem

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#392] [Q&A] [answered]
Hi all !
is anyone  sending a notice to validation url ("https://cvs.preview.ted.europa.eu/v1/notices/validation")   ?
I received this error : 
400 : "{"errorCode":400,"message":"The eForms SDK version 'eforms-sdk-1.6' does not exist.","timestamp":"2023-03-29T08:27:08Z"}"
Thanks a lot !
Renato

**Comment by elerik** [comment]
Yes same. Probably they are working on something. But it looks like it works again. I believe we can periodically expect this errors :)

**Comment by bertrand-lorentz** [comment, marked as answer]
There was indeed a new version of the validation service (CVS) deployed this morning.
The deployment is now completed, so things should be back to normal.

**Reply by rengit62** [reply]
Yes Bertrand, Thanks 
Renato
